osu.Game/Screens/Play/GameplayClockContainer.cs

```diff
@@ -200,11 +203,26 @@ public void StopUsingBeatmapClock()
         protected override void Update()
         {
             if (!IsPaused.Value)
+            {
                 userOffsetClock.ProcessFrame();
+            }
 
             base.Update();
         }
 
+        private double getTrueGameplayRate()
```

Comment:
This is unused, right?

Reply:
ah yep, this looks left over from before i moved it.